spec/cucumber/core/test/filters/locations_filter_spec.rb

```diff
@@ -102,20 +102,90 @@ module Core
                   | 1 | 2 |
                   | 3 | 4 |
 
+              Rule: A rule with a background
+                Background: A background rule
+                  Given background
+
+                Scenario: with a rule and background
+                  Given rule a
+
+                Scenario: another with a rule and background
+                  Given rule b
```

Comment:
shouldn't this also be rule a? I'm aware these are tests but just trying to keep things as compliant as poss